appveyor.yml

```diff
@@ -35,6 +35,7 @@ deploy:
     api_key:
       secure: 4yuwHXUFFa5ijOkkvvmEX/SjKhzWTvln7JtYrgVB1KdXcpF4o8WFAeNm6PTlwN+L
     skip_symbols: true # Will be uploaded by former deployment job
+    artifact: /^((?!MustardBlack\.Assets\.Nuglify)(.*))\.nupkg/
```

Comment:
this only seems to match "core" MB nupkg
![image](https://user-images.githubusercontent.com/1648478/86919960-e72fb880-c120-11ea-8e49-75502e48e344.png) - <https://regex101.com/r/i3JgNL/1>
I assumed you were trying to get it to upload all apart from nuglify

Reply:
my mistake. example link didn't have multiline on

Reply:
This works as expected when these flags are enabled
![image](https://user-images.githubusercontent.com/16237974/86920599-eba8a100-c121-11ea-8eb6-5378f0134975.png)
Can we include those flags on the `artifact` regex?